src/Compilers/Core/Portable/Diagnostic/DiagnosticSeverity.cs

```diff
@@ -27,7 +27,12 @@ public enum DiagnosticSeverity
         /// <summary>
         /// Something not allowed by the rules of the language or other authority.
         /// </summary>
-        Error = 3
+        Error = 3,
+
+        /// <summary>
+        /// Something that can be potentially improved.
+        /// </summary>
+        Suggestion = 4,
```

Comment:
If we already have a couple of severities that the command line compile doesn't care about, why does this concern you? :smile: The compiler API is 100% built in support of IDE experiences, so I'm not sure I understand the push back.

I find a design where Suggestion is a custom tag for Info to be enormously fragile. With that, we would unnecessarily create invalid combinations of severities and tags since "Suggestion" is nonsensical when combined with "Error" and "Warning". If Suggestion is upgraded to a new severity, the combinations of tags and severities are all sensible.

FWIW, the notion that UI presentation is controlled by tags is simply not reality today. We have one tag that controls UI presentation, but the _bulk_ of UI presentation is handled by severities (e.g. error and warning squiggles).